R/scoring.R

```diff
@@ -82,7 +82,7 @@ simulateNonGLM <- function(object, nsim, errorVec) {
   function(newdata) {
     fullErrors <- sample(errorVec, length(newdata)*nsim, replace = TRUE)
     newPreds <- as.vector(predict(object, newdata = newdata))
-    rep(newPreds, nsim) + fullErrors
+    lapply(rep(newPreds, nsim) + fullErrors, FUN = getScore)
```

Comment:
I don't think you need to do `rep` here. A scalar added to a vector will automatically be expanded.